src/lib/containers/UserCardListRotate.tsx

```diff
@@ -48,12 +50,15 @@ const UserCardListRotate: React.FunctionComponent<UserCardListRotateProps> = ({
     sql,
     count,
     token,
+    useQueryResultUserData = false,
+    size = 'LARGE USER CARD',
```

Comment:
Do you think we should use the constant `LARGE_USER_CARD` from 'lib/utils/SynapseConstants'?

Reply:
Yes